test/connection-session.test.ts

```diff
@@ -5,6 +5,10 @@ import authorize from './helper/webauth';
 import config from './config';
 import { isNodeJS } from './helper/env';
 
+if (typeof jest !== 'undefined') {
+  jest.retryTimes(2);
+}
+
```

Comment:
these auth tests launching a chromium instance sometimes fail (can't find the right HTML elements in the login page), rerunning it a few times help.

the `typeof` check is needed because this same file is loaded in a browser when running browser tests.